DOC-3566 multiple pgsql databases topic (#2883)
harness/developer-hub: docs/continuous-integration/use-ci/manage-dependencies/background-step-settings.md

Use these fields to define the commands that you need to run in the Background step.

### Shell

> Select the shell type for the commands defined in **Entry Point** or **Command**. Options include: **Bash**, **PowerShell**, **Pwsh** (PowerShell Core), **Sh**, and **Python**. If the step includes commands that aren't supported for the selected shell type, the build fails. Required binaries must be available on the build infrastructure or the specified image, as described in [Container Registry and Image](#container-registry-and-image).

Reviewer: ```suggestion
Select the shell type for the commands defined in **Entry Point** or **Command**. Options include: **Bash**, **PowerShell**, **Pwsh** (PowerShell Core), **Sh**, and **Python**. If the step includes commands that aren't supported for the selected shell type, the build fails. Required binaries must be available on the build infrastructure or the specified image. For more information, go to [Container Registry and Image](#container-registry-and-image).
```
